ghost.py

```diff
@@ -409,9 +424,6 @@ def init(self):
         dirname = os.path.dirname(self.db_path)
         if dirname and not os.path.isdir(dirname):
             os.makedirs(os.path.dirname(self.db_path))
-        elif os.path.isfile(self.db_path):
-            raise GhostError('Stash {0} already initialized'.format(
-                self.db_path))
 
```

Comment:
Well, that wouldn't be idempotent. I deleted it because we need to make sure that no matter which backend you use, running `init` twice would result in the same outcome.